### DISCUSSION TITLE
MY CHIA COINS HAVE DISAPPEARED FROM MY WALLET, AND I AM ALWAYS FARMING SOLO [BUG]

### DISCUSSION THREAD

**Discussion by verabeniel** [Chia-Network/chia-blockchain#6482] [Support]
I am farming solo since always and I earned two chias. These chias appear in my wallet until yesterday night when suddenly the wallet say 0 chias.

![Captura](https://user-images.githubusercontent.com/57694573/120897846-6d51c900-c628-11eb-81aa-c65080fceeeb.PNG)
![1Captura](https://user-images.githubusercontent.com/57694573/120897852-72af1380-c628-11eb-88ab-1db5ea537bef.PNG)

As you can see in the pictures , the history of my wallet only show the incomings transactions, and doesn´t show any outgoing transactions, so I don´t understand nothing. I´ve never been in any pool and never have downloaded any third party software.

In the chia explorer page, it appears like the two chias has spent at 384.137 height, but it isn´t true, I haven´t spent my chias.

This is the link : 

[https://www.chiaexplorer.com/blockchain/address/xch12x0ke62mery35jarwwnglkvvjsgg593sycuslpfsmnefw0avdtaqvu9gpn](url)

I need help please, I am so depressed!

Thanks in advance.

**Comment by sargonas** [comment]
Looking at the blockchain data, and your wallet address that I found when looking a the block your rewards came from, the chain shows your coins were spent on block 384,147 @ roughly 06/04/2021, 03:15:42 PM

You have found the link to the block with the transaction data in it, so I would see if you recognize the wallet address they were sent to and trace it back from there.

If you have no knowledge at all of this transaction, then it seems possible that someone has a copy of your security keys one way or another and accessed your wallet. In either case, this doesn't look like any kind of bug, but that the chia was directly transferred out of your wallet by someone with the security keys, the question now is just if this was done without your knowledge or not.

I would suggest generating a new wallet with fresh keys and moving over to that one if you are concerned about your keys being stolen, ideally on a fresh install if there is a risk of your current OS install being compromised.

**Reply by verabeniel** [reply]
But I only have the Chia Blockchain official client and nothing more than that.
I have never been in a pool and another guy has the same problem as me,
 and his chias appear spent too, at same time than mines, yesterday night and in my Chia Blockchain client only appear the transactions of incoming of when I earned the 2 chias. 
If someone has stolen me , shouldn`t it show in transactions history like an outgoing operation?
In another bug conversation at this page another people has the same problem and from yesterday night too at the same height 384137. I need help, I wasted too much money for the chia minning and this is devasting for me now.

**Reply by verabeniel** [reply]
I have seen that my chias has sent to this address without my consent (this only appears in chia explorer, my Chia client only show the incoming transactions). 
The guy has the same problem than me "spent" his chias too at same height, same hour and to the same address... I need a solution, then what kind of guarantees have the farmers of chia?? 
Because I only use the Chia Blockchain official client, nothing more never and I don`t deserve it.

![WhatsApp Image 2021-06-05 at 10 58 49 PM (1)](https://user-images.githubusercontent.com/57694573/120905398-211b7e80-c652-11eb-930d-2be55f944ad4.jpeg)
![WhatsApp Image 2021-06-05 at 10 58 49 PM](https://user-images.githubusercontent.com/57694573/120905399-21b41500-c652-11eb-995b-f9b95c58775d.jpeg)

**Reply by verabeniel** [reply]
![Address of my chias now](https://user-images.githubusercontent.com/57694573/120905538-0bf31f80-c653-11eb-9807-788a70547e90.PNG)

**Reply by sargonas** [reply]
There accident is a transaction history. If you look at the block in which the coins were transferred out, you will see a transaction call to transfer the coins from your respective wallets to another wallet. The fact they both happened at the exact same time would seem to reinforce my belief that it was a deliberate action taken by someone with keys to multiple wallets who was trying to “sweep” them.

It’s possible that you won’t see a transaction on your side because it’s likely when they used your keys to load the wallet on their side that they generated a new wallet public address that was tied to the same base wallet key, and initiated the transfer from there. 

I can assure you that the core fundamental technology in the Chia Blockchain is the same general concept and technology used in by various blockchains over the last decade, and it’s highly improbable there is a bug allowing non-transactional removal of coins from a wallet. All evidence so far indicates that this was clearly a case of compromised keys being used by a bad actor, if you did indeed have no agency over this transaction. 

It’s possible for keys to be compromised in a multitude of way even if you are using the official software, countless malware and viruses completely unrelated to chia will deep dive a users system and look for anything remotely resembling a password or security key or keyphrase  that is sent back home to a central server and applied to every known platform until a valid use for it is found.

**Reply by verabeniel** [reply]
In Reddit they say that the version 1.1.6 of the official chia blockchain client that you upload here on github was compromised !! So the problem was the official program and you don't offer any solution for all People have the same problem than me ...
I have been robbed because of the chia official client; Because many People have the same problem than me and they are farming solo too and have nothing more than Chia official blockchain client installed... so, in one hand the first version of Chia 1.1.6 official was compromised, you must know it , in Reddit they say that, or in another hand there is a fail of security of the 8444 port that it,s the only one I have open. And no one give me a solution !

**Reply by loppefaaret** [reply]
do you have a link to this report of 1.1.6 being compromised ?

**Comment by verabeniel** [comment]
The guy ave same problem than me says this to mw in the other bug conversation:

"it went to the same address: xch1cnm0qj7j9403yhs4wxqwqpk5cuku2n78lrtc3406nwlsmer2jrdsha9g7s
https://www.chiaexplorer.com/blockchain/address/xch1cnm0qj7j9403yhs4wxqwqpk5cuku2n78lrtc3406nwlsmer2jrdsha9g7s

whats funny is you and I are in the same block at the same time if it was stolen... without any transaction history, i dont think its stolen, something is wrong with the network randomly taking/spending coins from our wallets.. otherwise it would show the transactions in our wallets"

So how could Chia developers solve the problem?

**Comment by verabeniel** [comment]
Then with the address and puzzlehash information I gave to you , nobody can or are going to do nothing for protect us and restore our chias?
If there was a potential chia thief like you think , same person is stealing us, it`s is something very serious and we won`t be the only people affected.
It`s not fear.

**Comment by verabeniel** [comment]
The guy has the same problem than me has tested all things you say and
because of that he says that:

that is not true, in fact I have tested sending transactions with
generating new public key with same base wallet, regardless of what new
public key it is, the gui / cli will still show the transactions as long as
base wallet is same. So if this was a comprised wallet, and he restore with
new pub key in his end then transfer, it will not show in chiaexplorer but
it will definitely show in my wallet history. I suspected this and did
tested this myself.

Please we need support asap.
Thank you

**Reply by sargonas** [reply]
I understand your frustrations, but I assure you this is not something the Chia team can address for your directly. All evidence above clearly points to the likelihood that your keys were compromised. The lack of a transaction record on your local client could be due to multiple issues of a variety of causes, but that doesn't change the fact by looking at the block chain directly there is clearly a valid transaction that took place, something that *can not* happen without your private keys. Whomever transferred this XCH from your walet had your keys, if it was not you. 

I can sympathize with your frustration, but I assure you there is no evidence here of a code-level issue that Chia can address for you, as it seems to stem from an operational security related problem on user key security which is outside the scope of the kind of support Chia can provide.

**Comment by verabeniel** [comment]
The guy has the same problem than me has tested all things you say and because of that he says that: 


that is not true, in fact I have tested sending transactions with generating new public key with same base wallet, regardless of what new public key it is, the gui / cli will still show the transactions as long as base wallet is same. So if this was a comprised wallet, and he restore with new pub key in his end then transfer, it will not show in chiaexplorer but it will definitely show in my wallet history. I suspected this and did tested this myself.


Please we need support asap.

Thank you

**Comment by justinengland** [comment]
Hi. So if someone has your keys and did a transaction from their end. It might not be in logs on your local machine. If it's in the block chain explorer then it's on the blockchain usually. This is not a bug in the blockchain. This is an opsec concern. Someone harvested your keys. I would recommend using a new reward address. With new keys made on a clean os. Have you run any third party tools or scripts?

**Reply by verabeniel** [reply]
No, I never run any third party tool or script, I am always farming solo with the Chia Blockchain official client.

**Reply by room101-dev** [reply]
U could have visited a website, or gone anywhere logged your IP, the chia RPC is wide open "chia keys show --show-mneumonic" and its gone, poof

The only solution I have is to use hpool, and then create a cold wallet address, on a chia wallet offline machine and have your chia put into that address. 

If you run a 'chia farm and/or miner' on a public network, then your open to theft.

...

I got banned for ten days last weeks for posting this, so its obviously they don't like the truth. IMHO the Bangalore contractors who write the code in India, are harevesting the keys and the SV dev's don't care, they only care about the IPO.

**Reply by loppefaaret** [reply]
Where did you get banned for posting this ? the only solution is not hpool... please, come on ? you might deem hpool _your best_ choice, but it is not the only solution

**Comment by loppefaaret** [comment]
https://www.chiaexplorer.com/blockchain/address/xch12x0ke62mery35jarwwnglkvvjsgg593sycuslpfsmnefw0avdtaqvu9gpn
https://www.chiastatus.com/addresses/xch12x0ke62mery35jarwwnglkvvjsgg593sycuslpfsmnefw0avdtaqvu9gpn
i think chiaexplorer is having a db issue
they are now belonging to this address:
https://www.chiastatus.com/addresses/xch1cnm0qj7j9403yhs4wxqwqpk5cuku2n78lrtc3406nwlsmer2jrdsha9g7s

**Reply by verabeniel** [reply]
Then do you think I can recover my chias? In my wallet on Chia official  client don't show them, but in history are only incoming transactions, so its really weird. Until yesterday I have the two chias in my wallet. 
And now only appear in Farmed and in the History transactions.
More people have had the same problem than me and chias are sent to the same address and yesterday too.
I don`t knos what to think...

**Reply by loppefaaret** [reply]
only if that address belongs to your key, which i doubt, or your wallet would see them. they are gone and your key must _somehow_ have been compromised, you really need to figure out how that came to be - and if not able to, you should reinstall the operating system, and generate new key for "farming target reward" and direct any new wins to this new address, from a new key

**Reply by verabeniel** [reply]
But then, if my chias are disappear so easily and the chias of other people too, and to the same address that hasn't chia farmed and only has "chia transactions" of incoming chias he/she has stolen and will steal of many people when he wants;  how can I trust another time in Chia Blockchain officia? if  I continue farming, suposing i would be lucky another time , which i doubt, and i would get another two chias, they sure will disappear one day. There is no security for me. if I never be in a pool and never download any script and only farm solo and this happen to me, What guarantees I get for know this is not happen to me anymore? ITS HORRIBLE AND I AM VERY ANGRY AND DISSAPOINTED, BECAUSE I HAVE ONLY USED THE CHIA BLOCKCHAIN CLIENT OFFICIAL AND SEE WHATS HAPPEN...

**Reply by loppefaaret** [reply]
i am truely sorry that you have been robbed

**Reply by verabeniel** [reply]
I have been robbed because of the chia official client! Because many People have the same problem than me and they are farming solo too and have nothing more than Chia official blockchain client installed... so it's a fail of security of the 8444 port that it,s the only one I have open. And no one give me a solution !

**Comment by BrandtH22** [comment]
@verabeniel , unfortunately it appears someone has gained access to your keyset and/or device and stolen your funds. Since chia is a truly decentralized network there is no way to reclaim the funds or revert the transaction(s). One thing we recommend is ensuring that you and your farm are following security best practices to reduce malicious actors ability to gain access to your keys and/or device, many of these best practices are outlined here: https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

Since we unfortunately cannot do more to assist with the stolen funds we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
